docs/design/crd-validation/README.md

```diff
@@ -0,0 +1,79 @@
+## Background
+
+This proposal is to add support for admission controllers in OCS. An admission controller is a piece of code that intercepts requests to the Kubernetes API server prior to persistence of the object, but after the request is authenticated and authorized
+
+ There are two special controllers: MutatingAdmissionWebhook and ValidatingAdmissionWebhook.
+ 
+ Mutating controllers may modify the objects they admit but validation controllers are only allowed to validate requests.
```

Comment:
Do we need mutating controllers for OCS? I was assuming we just needed the validating controller to start with.

Reply:
Yes, thats correct we are implementing validatingwebhooks for the start. This is just a brief heads up on what type of admission controllers are available and what they do.